presto-docs/src/main/sphinx/connector/base-arrow-flight.rst

```diff
@@ -59,6 +59,33 @@ Property Name                               Description
 ``arrow-flight.server-ssl-enabled``         Port is ssl enabled
 ========================================== ==============================================================
 
+.. note::
+
+   A secure TLS connection is enabled whenever the server certificate
+   (``arrow-flight.server-ssl-certificate``) is provided, even if the
+   ``arrow-flight.server-ssl-enabled`` flag is not set.
+
+   If a client certificate (``arrow-flight.client-ssl-certificate``) is also provided,
+   the Flight client will be configured to perform mutual authentication.
+   However, a true mutual TLS (mTLS) connection is only established if the Flight server
+   is explicitly configured to **require and verify** client certificates.
+
```

Comment:
It would also be good to show a code snippet to set these options in the flight server